packages/flutter_tools/test/src/fakes.dart

```diff
@@ -252,6 +252,16 @@ class MemoryIOSink implements IOSink {
 
   @override
   Future<void> flush() async { }
+
+  void clear() {
+    writes.clear();
+  }
+
+  String getAndClear() {
```

Comment:
Adopted from https://github.com/flutter/flutter/blob/807bb298886a38971e59d9fcec86509fbafb4771/packages/flutter_tools/test/src/mocks.dart#L220-L224

I could be convinced that the `clear` method is unnecessary, but getting the string is convenient.